tests/components/owntracks/test_device_tracker.py

```diff
@@ -622,9 +622,12 @@ async def test_event_gps_entry_exit_wrong_order(hass, context):
 async def test_event_gps_entry_unknown_zone(hass, context):
     """Test the event for unknown zone."""
     # Just treat as location update
-    message = build_message({"desc": "unknown"}, REGION_GPS_ENTER_MESSAGE)
+    lat = INNER_ZONE["latitude"]
+    message = build_message(
+        {"desc": "unknown", "lat": lat, "acc": 40}, REGION_GPS_ENTER_MESSAGE
+    )
```

Comment:
Test modified because with new zone criteria the enter message did not make us enter the `inner` zone.